src/openslide-private.h

```diff
@@ -124,6 +124,8 @@ struct _openslide_format {
 };
 
 extern const struct _openslide_format _openslide_format_aperio;
+extern const struct _openslide_format _openslide_format_huron;
+extern const struct _openslide_format _openslide_format_dicom;
```

Comment:
Looks like this dicom line was accidentally copied in from the 4.0 branch - will remove it.